ingestion_server/ingestion_server/api.py

```diff
@@ -23,6 +26,15 @@
 CALLBACK_URL = "callback_url"
 SINCE_DATE = "since_date"
 
+sentry_sdk.init(
+    dsn=os.environ.get("SENTRY_DSN"),
+    integrations=[
+        FalconIntegration(),
+    ],
+    traces_sample_rate=os.environ.get("SENTRY_SAMPLE_RATE", 1.0),
```

Comment:
Would be interesting to know the results of the performance monitoring in Sentry!

Should we also add a TODO note to lower the `traces_sample_rate` as suggested in the docs?
https://docs.sentry.io/platforms/python/performance/#verify

Reply:
The ingestion process is internal so the number of events shouldn't be a problem here unlike in the public-faced services, I think!